src/base1/test-chan.js

```diff
@@ -254,11 +255,12 @@ QUnit.test("multiple", function (assert) {
     var channel = cockpit.channel({ host: "scruffy" });
     var channelb = cockpit.channel({ host: "amy" });
 
-    $(mock_peer).on("recv", function(event) {
+    const onrecv = event => {
         $(mock_peer).off("recv");
         assert.notStrictEqual(channel.id, channelb.id, "channels have different ids");
         done();
-    });
+    };
+    $(mock_peer).on("recv", onrecv);
```

Comment:
FTR, I split out the function as with standard event handlers it needs to have a name for `removeEventHandler()`. That doesn't hurt with jQuery, so it just makes the future porting easier.